### PR TITLE
bluetooth: tester: Added support for testing CSIS.

### PR DESCRIPTION
**Pull request by MSEEHenrik**
- Initialisation of Coordinated Set Identification Service.
- Set and clear member lock.
- Get member RSI.

### CODE REVIEW

File: tests/bluetooth/tester/src/btp_csis.c [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+static uint8_t csis_set_member_lock(const void *cmd, uint16_t cmd_len,
+				    void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_csis_set_member_lock_cmd *cp = cmd;
+	int err = -1;
+
+	if (csis_svc_inst) {
+		err = bt_csip_set_member_lock(csis_svc_inst, cp->lock, cp->force);
+	}
+	return (err) ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS;
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_csis.c:43]: Minor: We usually keep an empty line before `return` statements

File: tests/bluetooth/tester/src/btp_csis.c [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+static uint8_t csis_get_member_rsi(const void *cmd, uint16_t cmd_len,
+				   void *rsp, uint16_t *rsp_len)
+{
+	struct btp_csis_get_member_rsi_rp *rp = rsp;
+	int err = -1;
+
+	if (csis_svc_inst) {
+		err = bt_csip_set_member_generate_rsi(csis_svc_inst, rp->rsi);
+	}
+	rp->err = sys_cpu_to_le32(err);
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_csis.c:55]:
```suggestion
	}
	
	rp->err = sys_cpu_to_le32(err);
```
Minor: We usually keep an empty line after `}` unless that line also contains a `}`

File: tests/bluetooth/tester/src/btp_csis.c [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+	struct bt_csip_set_member_register_param register_params = {
+		.set_size = 1,
+		.set_sirk = { 0xB8, 0x03, 0xEA, 0xC6, 0xAF, 0xBB, 0x65, 0xA2,
+			      0x5A, 0x41, 0xF1, 0x53, 0x05, 0x68, 0x8E, 0x83 },
+		.lockable = true,
+		.rank = 1,
+		.cb = NULL
+	};
+	int err = bt_csip_set_member_register(&register_params, &csis_svc_inst);
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_csis.c:93]: Minor: We usually declare variables at top of scope`

File: tests/bluetooth/tester/src/btp_csis.c [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+		.func = csis_get_member_rsi
+	}
+};
+
+uint8_t tester_init_csis(void)
+{
+	tester_register_command_handlers(BTP_SERVICE_ID_CSIS, csis_handlers,
+					 ARRAY_SIZE(csis_handlers));
+
+	struct bt_csip_set_member_register_param register_params = {
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_csis.c:85]:
```suggestion
	const struct bt_csip_set_member_register_param register_params = {
```

File: tests/bluetooth/tester/src/btp_csis.c [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+		.set_size = 1,
+		.set_sirk = { 0xB8, 0x03, 0xEA, 0xC6, 0xAF, 0xBB, 0x65, 0xA2,
+			      0x5A, 0x41, 0xF1, 0x53, 0x05, 0x68, 0x8E, 0x83 },
+		.lockable = true,
+		.rank = 1,
+		.cb = NULL
+	};
+	int err = bt_csip_set_member_register(&register_params, &csis_svc_inst);
+
+	return (err) ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS;
```
Comment (Thalley) [inline tests/bluetooth/tester/src/btp_csis.c:95]: We should consider having a macro for `int err` to `BTP_STATUS`

**Review by Thalley — DISMISSED** [review]
Looks good - Some minor code style comments, but none that are necessary to fix IMO

File: tests/bluetooth/tester/src/btp_csis.c (hunk 1 above)
Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_csis.c]: Done.

File: tests/bluetooth/tester/src/btp_csis.c (hunk 2 above)
Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_csis.c]: Done,

File: tests/bluetooth/tester/src/btp_csis.c (hunk 3 above)
Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_csis.c:92]: Done.

File: tests/bluetooth/tester/src/btp_csis.c (hunk 4 above)
Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_csis.c]: Done.

File: tests/bluetooth/tester/src/btp_csis.c (hunk 5 above)
Comment (MSEEHenrik) [reply tests/bluetooth/tester/src/btp_csis.c]: Done.

**Review by Thalley — DISMISSED** [review]

**Review by sjanc — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**